src/App.js

```diff
@@ -1,24 +1,25 @@
 import axios from "axios";
 import { useState } from "react";
 import { useEffect } from "react";
-import "./App.css";
+import { Footer, Sidebar } from "./Components";
 
 function App() {
   const [someData, setSomeData] = useState([]);
 
   useEffect(() => {
     (async function () {
-      const { data } = await axios.get("/api/brands");
-      setSomeData(data.brands);
+      const { data } = await axios.get("/api/products");
+      setSomeData(data.products);
       console.log(data);
```

Comment:
Remove console.log

Reply:
yes this app.js is just for testing purposes it will be cleared later